src/inputgroup/skin.css

```diff
@@ -86,3 +86,89 @@
     }
   }
 }
+
+/* 
+  TODO: discuss with other 
+  Way to keep this datapicker related
+  IN PROGRESS
+*/
+.spectrum-InputGroup {
+  &.spectrum-Datepicker {
+    /** Normal **/
+    &:not(.spectrum-InputGroup--quiet) {
+      border: 1px solid var(--spectrum-textfield-border-color);
+    }
+    /** Quiet **/
+    &.spectrum-InputGroup--quiet {
+      border-bottom: 1px solid var(--spectrum-textfield-border-color);
+      hr {
+        left: calc(50% - 21px); /*22px half button width and 5px hald of HR width*/
```

Comment:
should not use pixel values or it won't scale properly. Be sure to test locally by swapping themes and scales in order to see this in action (note the location of the <hr>):
![image](https://user-images.githubusercontent.com/13972198/49836829-64d2ed00-fd61-11e8-94e4-1822e45890f0.png)